src/vaadin-list-box.html

```diff
@@ -91,11 +91,23 @@
         ready() {
           super.ready();
           this.setAttribute('role', 'list');
+
+          this._debouncerCheckImport = Polymer.Debouncer.debounce(
```

Comment:
The `ready()` will just be called once so there's no point in using a debouncer. Normal `setTimeout()` of 2000ms works fine here

Reply:
Good point, should we still be aligned with the `vaadin-grid` in terms of having `_checkImport` method or would it be better to move it inside timeout? 🤔

Reply:
Separate checkImport is fine imo